resources/home/dnanexus/generate_fusion_workbook.py

```
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# eggd_generate_fusion_workbook 1.0.0
+
+
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)